<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lDesdeLaMuestra">
         <property name="text">
          <string>Duracion (Seg)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lCantMuestrasGraficadas">
         <property name="text">
          <string>Frecuencia Muestreo (Ej: 44100 hz)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textDuracion">
         <property name="minimumSize">
          <size>
           <width>225</width>
           <height>42</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>225</width>
           <height>42</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textFrecuencia">
         <property name="minimumSize">
          <size>
           <width>225</width>
           <height>42</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>225</width>
           <height>42</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="7">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="6">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="pbGraficar_Tiempo">
     <property name="text">
      <string>Grafico en Tiempo</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="pbGraficar_Espec">
     <property name="text">
      <string>Espectrograma</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pbGraficar_Frec">
     <property name="text">
      <string>Grafico en Frecuencia</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6" colspan="2">
    <widget class="QLabel" name="pantalla">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbComArduino">
       <property name="text">
        <string>Com Arduino / Grabacion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbReproducir">
       <property name="text">
        <string>Reproducir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Valor Detectado</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
